components/net/image_cache.rs

```diff
@@ -566,9 +676,111 @@ impl ImageCache for ImageCacheImpl {
         }
     }
 
+    fn add_rasterization_complete_listener(
+        &self,
+        pipeline_id: PipelineId,
+        image_id: PendingImageId,
+        size: DeviceIntSize,
+        sender: IpcSender<ImageCacheMessage>,
+    ) {
+        let mut store = self.store.lock().unwrap();
+        let Some(_) = store.vector_images.get(&image_id).cloned() else {
```

Comment:
use `contains_key`?

Reply:
Yep, this was an oversight from a previous refactoring. Fixed in [d6c9182](https://github.com/servo/servo/pull/36721/commits/d6c91826b4885176da8fc5946580d4eea2f0b9ce).